src/main/java/twilightforest/world/components/feature/templates/DruidHutFeature.java

```diff
@@ -53,7 +53,7 @@ protected void postPlacement(WorldGenLevel world, RandomSource random, Structure
 			config.buildAddProcessors(placementSettings, random);
 			placementSettings.addProcessor(CobbleVariants.INSTANCE).addProcessor(StoneBricksVariants.INSTANCE);
 
-			template.placeInWorld(world, placementPos, placementPos, placementSettings, random, 20);
+			template.placeInWorld(world, placementPos, placementPos, placementSettings, random, Block.UPDATE_CLIENTS);
```

Comment:
20 looks to be UPDATE_KNOWN_SHAPE and UPDATE_INVISIBLE, are these still needed?

Reply:
Drullkus said it is fine to place without them if I think so.
[Proof](https://discord.com/channels/313006291012288521/384911556468801537/1290035483153535057)